go.mod

```diff
@@ -3,7 +3,6 @@ module github.com/hashicorp/boundary-plugin-aws
 go 1.21
 
 require (
-	github.com/aws/aws-sdk-go v1.44.80
 	github.com/aws/aws-sdk-go-v2 v1.20.1
```

Comment:
Do we want to update this version?

Reply:
Will do this as a separate PR after https://github.com/hashicorp/go-secure-stdlib/pull/128 is merged 👍